docs/intro/tutorial03.txt

```diff
@@ -66,6 +66,9 @@ slightly different, because they take an argument:
 
 .. snippet::
     :filename: polls/views.py
+    
+    from django.http import HttpResponse
```

Comment:
This import was already added in tutorial 1. The instructions say to *add* these views.